tests/test.yml

```diff
@@ -182,6 +182,16 @@ samba_shares:
     browsable: yes
     path: "{{ samba_shares_root }}/photos"
 
+###
```

Comment:
Please move and update this section as per `all.yml.dist`

Reply:
Done.